dataminer/Professional_services/Training_courses_and_certification/DataMiner_Connector_Integration_Initiation.md

```diff
@@ -0,0 +1,51 @@
+---
+uid: DataMiner_Connector_Integration_Initiation
+---
+
+# DataMiner Connector Integration: Initiation Training
+
+Building a connector may sound straightforward, but getting it right requires a solid foundation. The Initiation Training is your starting point.
+
+This training equips you with the **fundamental skills to design and build DataMiner connectors**, combining theory with extensive hands-on practice. Through exercises on protocol structure, UI components, device data handling, QActions, and traps, you will learn how to create functional connectors that integrate smoothly into DataMiner.
+
+## What you will learn
+
+### Competencies
+
+By the end of the training, you will have developed the ability to:
+
+- Understand and apply DataMiner connector protocol structure.
+- Design and configure UI components.
+- Work with device data using SNMP standalone parameters, flows, and tables.
+- Write and debug QActions for logic implementation.
+- Manage traps effectively.
+
+### Goals
+
+At the end of this course, you will be able to:
+
+- Build a functional DataMiner connector covering UI, data handling, and logic.
+- Apply QActions and traps in practical scenarios.
+- Confidently take the next step toward advanced integration (mastership).
+
+## Training content
+
+| Day | AM Session | PM Session |
+|--|--|--|
+| Monday | 01.01 - Introduction <br> 01.02 - Protocol Structure <br> 01.03 - DIS | 01.04 - Basic UI Components <br> 01.04 - Exercise: UI + Logic |
+| Tuesday | 01.04 - Exercise (continued) | 01.05 - Working With Device Data – SNMP Standalone Params <br> 01.05.1 - Exercise: SNMP Standalone Params |
+| Wednesday | 01.05 - Working With Device Data – Flow + SNMP Tables | 01.05.1 - Exercise: SNMP Standalone Params <br> 01.05.2 - Exercise: SNMP Tables |
+| Thursday | 01.06 - QActions | 01.06 - Exercise: QActions |
+| Friday | 01.07 - Traps | 01.07 - Exercise: Traps |
+
+## Practical info
+
+- **Duration**: 5 days.
+- **Format**: On-site (hands-on, highly recommended).
```

Comment:
This isn't entirely clear to me. What do you mean with "hands-on, highly recommended"? Since no other alternative than hands-on training is given, I'm not sure if it makes sense to add this. (Same for the Mastership page).